### DISCUSSION TITLE
Text posts in the Feed.

### DISCUSSION THREAD

**Discussion by zarkones** [community/community#17440] [Other Feature Feedback, Questions, & Ideas]
GitHub allows people to follow other users of the platform.
But there is a barrier on how to communicate information with the followers.
Maybe the feed should include texts, code snippets, links and images posted by the community.
Something like on Twitter.

**Comment by siminapasat** [comment]
Hi @zarkones 👋 PM at GitHub here 😊 

What kind of information would you like to share with your followers? Is this from you personally or on behalf of some of the (open source) projects you might maintain?

**Reply by zarkones** [reply]
GitHub already allows me to share my knowledge and work with the world.
I would like to share my thinking with the world on behalf of myself or on the behalf of (open source) project.
Something casual in a way or formal.
I think it should be up to creators to decide to whom they're referring to and with which message.
Like making a pool asking people which feature would the like to be implemented next.
Or it could be as simple as just saying "hello, how was your day?".
If I post something, I would appreciate if people would be able to up-vote it, comment, share, etc...
I hope this clears things up.
